Lib/ufo2ft/outlineCompiler.py

```diff
@@ -1495,11 +1482,9 @@ def setupOtherTables(self):
 
         self.setupTable_glyf()
 
-        if self.ufo.info.openTypeGaspRangeRecords:
-            self.setupTable_gasp()
-
         self.instructionCompiler.setupTable_cvt()
         self.instructionCompiler.setupTable_fpgm()
+        self.instructionCompiler.setupTable_gasp()
```

Comment:
this method used to be guarded by `if "gasp" not in self.tables:`, now it's done unconditionally.
the OutlineCompiler.tables lists the table tags that caller requests to build, we should abide by this list before we call each respective `setupTable_{tag}`. @jenskutilek can you please do that?

Reply:
Done.